ivy/functional/frontends/tensorflow/linalg.py

```diff
@@ -10,6 +10,10 @@ def det(input, name=None):
     return ivy.det(input)
 
 
+def eigvals(tensor, name=None):
+    return ivy.eigvals(tensor)
```

Comment:
This function is not present in `Ivy`. It is present in our Extended API task.
You can work on it to implement it.
Thanks!
https://github.com/unifyai/ivy/issues/3856